include/RE/U/Utility.h

```diff
@@ -0,0 +1,558 @@
+#pragma once
+
+
+
+
```

Comment:
this entire file really needs some more thinking and formatting (and namespacing)